content/docs/concepts/architecture/index.md

```diff
@@ -9,73 +9,137 @@ Keptn is an event-based control plane for continuous delivery and automated oper
+The shared secret is generated during the installation and verified by the *api* component.
 
 ## Keptn Bridge
 
-The *Keptn Bridge* provides a user interface that shows all Keptn-managed projects and services. Besides, it gives an overview of the staging environment. 
+The *Keptn Bridge* provides a user interface that shows all projects and services that Keptn manages
+and gives an overview of the staging environment. 
 
-**Note:** A dedicated section for the Keptn Bridge is provided [here](../../0.15.x/bridge/), which explains how to access it and shows the user interface.
+See [Keptn Bridge](../../0.18.x/bridge/),
+for information about how to access and use the Keptn Bridge.
 
 ## Keptn Control Plane
 
-Keptn has several core components that are set up during the installation.
+The Keptn Control Plane runs the basic components that are required
+to run Keptn and to manage projects, stages, and services.
+This includes handling events and providing integration points.
+It orchestrates the task sequences that are defined in the *shipyard*
+but does not actively execute the tasks.
 
 ### api-gateway-nginx
 
-The *api-gateway-nginx* component is the single point used for exposing Keptn to the outside world. Besides, it is responsible to redirect incoming requests to the right internal Keptn endpoints, e.g., to api, bridge, or configuration-service.
+The *api-gateway-nginx* component is the single point used for exposing Keptn to the outside world.
+It supports the four [access options](../../install/access) that Kubernetes supports:
+LoadBalancer, NodePort, Ingress, and Port-Forward.
+
+It also redirects incoming requests to the appropriate internal Keptn endpoints --
+api, bridge, or configuration-service.
 
 ### api-service
 
-The *api* component provides a REST API that allows communicating with Keptn. It provides endpoints to authenticate, get metadata about the Keptn installation within the cluster, forwarding CloudEvents to the NATS cluster, and triggering evaluations for a service.
+The [Keptn API](../../0.18.x/reference/api/) provides a REST API
+that allows you to communicate with Keptn.
+It provides endpoints to authenticate, get metadata about the Keptn installation within the cluster,
+forward CloudEvents to the NATS cluster, and triggering evaluations for a service.
 
-**Note:** A dedicated Keptn API section is provided [here](../../0.15.x/reference/api/), which helps you to access the API and to explore the available endpoints.
+The documentation shows you how to access the API and to explore the available endpoints.
 
 ### mongodb-service
 
-The *mongodb-datastore* provides means to store event data in a MongoDB deployed in your Keptn namespace. Hence, the service provides the REST endpoint `/events` to query events.
+The *mongodb-datastore* stores event data in a MongoDB deployed in your Keptn namespace.
+Hence, the service provides the REST endpoint `/events` to query events.
 
 ### configuration-service
```

Comment:
As far as I know, the configuration service does not exist anymore in newer keptn versions and the successor is the resource service

Reply:
Good catch!    @thisthat I tried to do a little fix here based on the info in the 16.0.x Release Notes but it looks like the resource-service was in Keptn as a Beta feature from 0.12.x forward.   The only other mentions I find in the docs are in the reference page for keptn_add_resource and that isn't a lot of information.  Please advise.